tests/test_generate.py

```diff
@@ -337,24 +337,19 @@ def test_generate(yaml_dir, tmpdir):
     # Verify old version HTML is not updated.
     html_blob = bucket.get_blob(html_blob.name)
     t9 = html_blob.updated
-    assert t8 == t9
+    assert t8 == t9, "old version not updated"
 
     # Verify latest version HTML is updated.
     latest_html_blob = bucket.get_blob(latest_html_blob_name)
     t2_latest = latest_html_blob.updated
-    assert t1_latest != t2_latest
+    assert t1_latest != t2_latest, "latest is updated"
 
     # Update the YAML, build new docs, and verify the old HTML is updated.
     upload_yaml(yaml_dir, test_bucket)
     generate.build_new_docs(test_bucket, storage_client)
     html_blob = bucket.get_blob(html_blob.name)
     t10 = html_blob.updated
-    assert t9 != t10
-
-    # Also verify the latest HTML is updated.
-    latest_html_blob = bucket.get_blob(latest_html_blob_name)
-    t3_latest = latest_html_blob.updated
-    assert t2_latest != t3_latest
```

Comment:
Would it be worth checking if the latest blob is not updated?

Reply:
🙈 -- this can be ignored as it's been merged!

Reply:
I think it's fine to have it removed - since we already checked with `assert t9 != t10`; `t2_latest != t3_latest` just checks that the timestamp has indeed been updated but `t10 == t3_latest` in this case.